README.md

```diff
@@ -18,6 +18,29 @@ See [Component Documentation](https://sap.github.io/fundamental-react/) for exam
 You will need to install [Node and Node Package Manager](
 https://www.npmjs.com/get-npm).
 
+## Consuming the Library
+
+1. Fundamental-react does not include the SAP Fiori Fundamentals library which is required for styling. Install Fiori Fundamentals:
+    ```
+    npm install fiori-fundamentals
+    ```
```

Comment:
This first point is actually not accurate.  Is _does_ include this as a dependency.